app/assets/stylesheets/custom/govuk-table__foot.scss

```diff
@@ -0,0 +1,21 @@
+.govuk-table .govuk-table__foot {
```

Comment:
Can be written as: `.govuk-table {
	.govuk-table__foot {
		font-weight: 700;
		border-top: 3px solid #b1b4b6;
		.govuk-table__row {
			.govuk-table__footer {
				text-align: left;
				padding: 10px 20px 10px 0;
				&:last-child {
					padding-right: 0;
				}
			}
			.govuk-table__footer--numeric {
				text-align: right;
			}
			&:last-of-type {
				.govuk-table__header {
					border-bottom: none;
				}
			}
		}
	}
}`

Reply:
.govuk-table {
	.govuk-table__foot {
		font-weight: 700;
		border-top: 3px solid #b1b4b6;
		.govuk-table__row {
			.govuk-table__footer {
				text-align: left;
				padding: 10px 20px 10px 0;
				&:last-child {
					padding-right: 0;
				}
			}
			.govuk-table__footer--numeric {
				text-align: right;
			}
			&:last-of-type {
				.govuk-table__header {
					border-bottom: none;
				}
			}
		}
	}
}

Reply:
`.govuk-table {
	.govuk-table__foot {
		font-weight: 700;
		border-top: 3px solid #b1b4b6;
		.govuk-table__row {
			.govuk-table__footer {
				text-align: left;
				padding: 10px 20px 10px 0;
				&:last-child {
					padding-right: 0;
				}
			}
			.govuk-table__footer--numeric {
				text-align: right;
			}
			&:last-of-type {
				.govuk-table__header {
					border-bottom: none;
				}
			}
		}
	}
}`